tests/api/nb_ngram_api/test_make_word_graph.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import unittest.mock
+from dhlab.api.nb_ngram_api import make_word_graph
+import json
+import networkx as nx
+from typing import Literal
+
+
+def _links_to_edge_view(nodes: list[str], links: list[dict[str, int]]) -> dict[tuple[str, str], dict[Literal["value", int]]]:
```

Comment:
```suggestion
def _links_to_edge_view(nodes: list[str], links: list[dict[str, int]]) -> dict[tuple[str, str], dict[Literal["weight"], int]]:
```

Fixed some type-hinting typos.